site/lib/src/components/tutorial/client/progress_ring.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2025 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math';
+
+import 'package:jaspr/jaspr.dart';
+
+class InteractiveProgressRing extends StatelessComponent {
```

Comment:
There is no actual interactivity right now, but having a separate component here means its easy to put a `@client` on it in the future and add some logic.